src/games/strategy/triplea/ai/proAI/ProData.java

```diff
@@ -0,0 +1,43 @@
+package games.strategy.triplea.ai.proAI;
+
+import games.strategy.engine.data.GameData;
+import games.strategy.triplea.Properties;
+
+/**
+ * Pro AI data.
+ */
+public class ProData {
+
+  public static double winPercentage = 95;
```

Comment:
Ah, they are initial values, setting these values in the constructor is more expected.